code/users/plugin.js

```diff
@@ -0,0 +1,113 @@
+const boom = require("boom");
+const modelUtils = require("../model-utils");
+const schema = require("./schema");
+
+module.exports = {
+  name: "users",
+  version: "1.0.0",
+  async register(server) {
+    server.route({
+      method: "POST",
+      path: "/",
+      options: {
+        tags: ["api"],
+        response: { schema: schema.get },
+        validate: {
+          payload: schema.create
+        }
+      },
+      handler: async (request, h) => {
```

Comment:
What do you think about the concept of separating these handlers into another export, so they're not all anonymous functions?

Reply:
It's a fine concept, but for tiny CRUD handlers I actually find the maintenance much easier when I can grep the code for the API path and edit that file and be done. If the handlers get larger, sure split them out to separate named functions or even separate modules, but I start with defining them inline. My experience with projects where everything is split out too much is navigating around to the right file gets tedious.

Reply:
Makes perfect sense.